File: docs/src/testers/sessions/cdp/selenium_playwright_alternative.ts
// @sniptest filename=selenium_playwright_alternative.ts
// @sniptest show=1-12
import { NotteClient } from 'notte-sdk';

const client = new NotteClient();

const status = await client.Session().use(async session => {
  const status = await session.status();
  const page = await session.page();
  // Use Playwright for automation
  await page.goto('https://example.com');
  console.log(`Title: ${await page.title()}`);
  return status;
});

export { status };

File: docs/src/testers/sessions/cdp/selenium_playwright_alternative.py
# @sniptest filename=selenium_playwright_alternative.py
# @sniptest show=1-11
from notte_sdk import NotteClient

client = NotteClient()

with client.Session() as session:
    # Access the Playwright page directly
    page = session.page

    # Use Playwright for automation
    page.goto("https://example.com")
    print(f"Title: {page.title()}")
    status = session.status()
